Updated translations for English and Spanish

juliomartinezrodenas opened the pull request:
Hello,

With all the new features this app is now easier to recommend, since the only real competition it had was Weather Timeline and it cannot be downloaded for new users.
- I have updated the Spanish translations for version 2.302. The strings.xml is a bit messy because I've used Android Studio's Translator Editor. I hope that it's alright with you
- I have changed a few strings in the English version that were phrased a bit weird. 
- I have also changed the capitalization of strings to make it more standard, and removed many final periods (.) since they usually look weird when designing labels, alerts, toasts and other UI components

I'd like these changes to be merged into the next build but I'm open to criticism.

Thank you and good job on this app

WangDaYeeeeee commented:
Thanks for your work. I will check it as soon as possible.

juliomartinezrodenas <notifications@github.com> 于 2019年6月18日周二 01:09写道：

> Hello,
>
> With all the new features this app is now easier to recommend, since the
> only real competition it had was Weather Timeline and it cannot be
> downloaded for new users.
>
>    - I have updated the Spanish translations for version 2.302. The
>    strings.xml is a bit messy because I've used Android Studio's Translator
>    Editor. I hope that it's alright with you
>    - I have changed a few strings in the English version that were
>    phrased a bit weird.
>    - I have also changed the capitalization of strings to make it more
>    standard, and removed many final periods (.) since they usually look weird
>    when designing labels, alerts, toasts and other UI components
>
> I'd like these changes to be merged into the next build but I'm open to
> criticism.
>
> Thank you and good job on this app
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/WangDaYeeeeee/GeometricWeather/pull/93
> Commit Summary
>
>    - Updated translations for English and Spanish
>
> File Changes
>
>    - *M* app/src/main/res/values-es/arrays.xml
>    <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93/files#diff-0>
>    (47)
>    - *M* app/src/main/res/values-es/strings.xml
>    <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93/files#diff-1>
>    (207)
>    - *M* app/src/main/res/values/arrays.xml
>    <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93/files#diff-2>
>    (4)
>    - *M* app/src/main/res/values/strings.xml
>    <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93/files#diff-3>
>    (188)
>
> Patch Links:
>
>    - https://github.com/WangDaYeeeeee/GeometricWeather/pull/93.patch
>    - https://github.com/WangDaYeeeeee/GeometricWeather/pull/93.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93?email_source=notifications&email_token=AD2VOYWANPAUAVQYZDIT44TP27AODA5CNFSM4HYYTBQ2YY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4GZ562VQ>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD2VOYT32TAL7SOWIOARCUTP27AODANCNFSM4HYYTBQQ>
> .
>

juliomartinezrodenas commented:
I will make another pull request over the weekend. There are a couple texts
I'm not entirely happy about. You can reject this one

El vie., 21 jun. 2019 7:16, wangdaye <notifications@github.com> escribió:

> Thanks for your work. I will check it as soon as possible.
>
> juliomartinezrodenas <notifications@github.com> 于 2019年6月18日周二 01:09写道：
>
> > Hello,
> >
> > With all the new features this app is now easier to recommend, since the
> > only real competition it had was Weather Timeline and it cannot be
> > downloaded for new users.
> >
> > - I have updated the Spanish translations for version 2.302. The
> > strings.xml is a bit messy because I've used Android Studio's Translator
> > Editor. I hope that it's alright with you
> > - I have changed a few strings in the English version that were
> > phrased a bit weird.
> > - I have also changed the capitalization of strings to make it more
> > standard, and removed many final periods (.) since they usually look
> weird
> > when designing labels, alerts, toasts and other UI components
> >
> > I'd like these changes to be merged into the next build but I'm open to
> > criticism.
> >
> > Thank you and good job on this app
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/WangDaYeeeeee/GeometricWeather/pull/93
> > Commit Summary
> >
> > - Updated translations for English and Spanish
> >
> > File Changes
> >
> > - *M* app/src/main/res/values-es/arrays.xml
> > <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93/files#diff-0>
> > (47)
> > - *M* app/src/main/res/values-es/strings.xml
> > <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93/files#diff-1>
> > (207)
> > - *M* app/src/main/res/values/arrays.xml
> > <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93/files#diff-2>
> > (4)
> > - *M* app/src/main/res/values/strings.xml
> > <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93/files#diff-3>
> > (188)
> >
> > Patch Links:
> >
> > - https://github.com/WangDaYeeeeee/GeometricWeather/pull/93.patch
> > - https://github.com/WangDaYeeeeee/GeometricWeather/pull/93.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/WangDaYeeeeee/GeometricWeather/pull/93?email_source=notifications&email_token=AD2VOYWANPAUAVQYZDIT44TP27AODA5CNFSM4HYYTBQ2YY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4GZ562VQ
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AD2VOYT32TAL7SOWIOARCUTP27AODANCNFSM4HYYTBQQ
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WangDaYeeeeee/GeometricWeather/pull/93?email_source=notifications&email_token=AHYVKBLOLDKOJC6K47KXL4LP3RP4DA5CNFSM4HYYTBQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODYHOKVI#issuecomment-504292693>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHYVKBNXD7UFHEUQTNWVR7LP3RP4DANCNFSM4HYYTBQQ>
> .
>
